apps/omg_watcher/lib/api/status.ex

```diff
@@ -51,7 +50,7 @@ defmodule OMG.Watcher.API.Status do
         last_mined_child_block_number: last_mined_child_block_number,
         last_mined_child_block_timestamp: last_mined_child_block_timestamp,
         eth_syncing: Eth.Geth.syncing?(),
-        byzantine_events: prepare_events(events_processor ++ events_blockgetter)
+        byzantine_events: events_processor ++ events_blockgetter
```

Comment:
It's not something about your changes, but since we are in this file, two things can be fixed:
1. You can be more specific in defining type t(), byzantine_events is of type list(OMG.Watcher.Event.t())
2. events_blockgetter -> events_block_getter